src/Option/Option.ts

```diff
@@ -142,6 +142,29 @@ export const map = <A extends NonNullish, B extends NonNullish>(
         none: none,
     })
 
+/**
+ * Execute an arbitrary side-effect function if `Some`.
```

Comment:
🎨 Might be worth noting in the remarks that this is a terminal function in a `pipe`line

Reply:
Updated!